frontend/tailwind.config.ts

```diff
@@ -1,80 +1,97 @@
-import type { Config } from "tailwindcss"
+import type { Config } from 'tailwindcss'
 
 const config = {
-  darkMode: ["class"],
+  darkMode: ['class'],
   content: [
     './pages/**/*.{ts,tsx}',
     './components/**/*.{ts,tsx}',
     './app/**/*.{ts,tsx}',
     './src/**/*.{ts,tsx}',
-	],
-  prefix: "",
+  ],
+  prefix: '',
   theme: {
+    screens: {
+      'xs': '250px',
+      'sm': '500px',
```

Comment:
I would recommend we use 320px, 480px, 768px, 1024px and 1200px breakpoints. These standard breakpoints often fall within the range of common device widths—typically those of smartphones, tablets, and desktops. Can explore if you can find more appropriate set.